.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 2
+indent_style = space
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.{md,mdx}]
+trim_trailing_whitespace = false
```

Comment:
Interesting, why explicitly allow trailing whitespace in markdown files?

Reply:
Great question and I'm gonna remove this (I copied from another project and missed this when reviewing)